improver/utilities/threshold_interpolation.py

```diff
@@ -118,11 +113,9 @@ def _interpolate_thresholds(
             forecast_at_interpolated_thresholds
         )
 
-        # Reshape forecast_at_percentiles, so the percentiles dimension is
-        # first, and any other dimension coordinates follow.
```

Comment:
You've deleted this comment rather than rewritten it to explain what is actually happening here. Can you write it again but correct it to describe what this function is doing to your data.